### PR TITLE
PoC: allow to run ConsoleApp in net452 and net46

### PR DESCRIPTION
**Pull request by pjanotti**
Small change to allow to target .NET Framework with the ConsoleApp, the server app is .NET Core so for now, only the client can be on Framework.

### CODE REVIEW

**Comment by pjanotti** [comment]
@pellared on Ubuntu 20.04 WSL2 I have to define `CURDIR` prior to run the script, otherwise, I get: 
```terminal
$ envvars.sh: line 26: CURDIR: unbound variable
```

I tried `set +u` before sourcing the script but that didn't fix the issue.

**Comment by pellared** [comment]
> @pellared on Ubuntu 20.04 WSL2 I have to define `CURDIR` prior to run the script, otherwise, I get:
> 
> ```
> $ envvars.sh: line 26: CURDIR: unbound variable
> ```
> 
> I tried `set +u` before sourcing the script but that didn't fix the issue.

I guess that changing 
https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/blob/ea124dcec06bc15522999ac7b2967454a7980a77/dev/envvars.sh#L26

to 

```
if [[ -z "${CURDIR:-}" ]]; then
```

will solve the issue. Probably I have forgotten to test it on Linux before pushing 👎

**Comment by pjanotti** [comment]
```bash
if [[ -z "${CURDIR:-}" ]]; then
```
Nice fix 😃 I feel dumb for not coming up with it!

**Review by dszmigielski — APPROVED** [review]
LGTM

File: poc.sh [hunk 1]
```diff
@@ -28,10 +29,10 @@ docker run -d --rm --name jaeger \
 
 # instrument and run HTTP server app in background
 ASPNETCORE_URLS="http://127.0.0.1:8080/" ./dev/instrument.sh dotnet run --no-launch-profile -f $publishTargetFramework -p ./test/test-applications/integrations/Samples.AspNetCoreMvc31/Samples.AspNetCoreMvc31.csproj &
-sleep 10 # wait until it starts
+./dev/wait-local-port.sh 8080
```
Comment (pellared) [inline poc.sh:32]: ❤️

**Review by pellared — APPROVED** [review]

**Pull request merged**